### ISSUE TITLE
Suggestion: Symmetrical Powerups

### ISSUE THREAD

**Issue by rabbatypus** [tagpro-game/tagpro-issues#191]
Currently, the general consensus is that with the way powerups are, powerup spawns will be fair in the long run. I don't think this is really a good system because competitive matches are based off of twenty minutes of gameplay which, with perfect efficacy, only allows for twenty spawns for each powerup. This is not a large enough sample to generate 1/3 spawn rate for each powerup dependably, i.e. very often, a spawn will generate significantly more of one powerup than the other two. This is a real problem. For example, on Boombox, a team that starts with a TagPro is at a massive advantage against a team that starts with a Juke Juice.

My suggestion is simple. Firstly, maps should be coded so that symmetric powerups are linked together. Then, when a match is started, the server will create a string that is n characters long for each set of symmetric powerups, where n = minutes in the match. Powerups will spawn by following their string. For example, the team powerups on Boombox will both spawn a TagPro on their third time spawning, but when this occurs depends on the efficacy of the two teams. This will make it so that not only are powerups still random but also the only factor in determining who gets a powerup advantage is the skill of both teams.

I understand that this suggestion has a few unintended consequences. For example, sticking with Boombox, a team whose powerup spawns sooner will also be able to predict the next powerup that spawns on the other side of the map, allowing them to decide whether or not they want to fight for it. However, I think that consequences like this can actually be good by encouraging a more strategic assessment of when a team wants to collect a powerup instead of constantly forcing teams to take a player out of action just so that the other team won't get a powerup.

Such consequences may be seen as issues, but if they are, I think they outweigh the disbalance created by the current powerup system.

**Comment by rabbatypus** [comment]
I made a shitty "visualization" of my suggestion. I don't know how to link images, so here's the link: http://i.imgur.com/itZgLiK.png.

I've imagined it as feeding a string of powerups into the powerup spawn. Notice that the two opposite powerups have identical strings. Again, the only factor affecting when a team gets which powerup is their efficacy in obtaining powerups. This allows for both random and equal games.
